testsuite/cluster-test/src/experiments/versioning_test.rs

```diff
@@ -107,18 +108,21 @@ impl Experiment for ValidatorVersioning {
         // TODO: In the future we may want to pass this functor as an argument to the experiment
         // to make versioning test extensible.
         let txn_payload = TransactionPayload::ScriptFunction(ScriptFunction::new(
-            ModuleId::new(account.address, Identifier::new("PHANTOM_MODULE").unwrap()),
+            ModuleId::new(
+                account.address(),
+                Identifier::new("PHANTOM_MODULE").unwrap(),
+            ),
             Identifier::new("PHANTOM_FUNCTION").unwrap(),
             vec![],
             vec![],
         ));
 
-        let txn_gen = |account: &AccountData| {
+        let txn_gen = |account: &LocalAccount| {
             create_user_txn(
-                &account.key_pair,
+                &account.key_pair(),
                 txn_payload.clone(),
-                account.address,
-                account.sequence_number,
+                account.address(),
+                account.sequence_number(),
                 123456,
```

Comment:
Change this to use a TransactionFactory + Local account directly instead of using the `create_user_txn` function